eo-runtime/src/test/java/EOorg/EOeolang/EOio/EOstdinTest.java

```diff
@@ -54,13 +58,33 @@ public final class EOstdinTest {
 
     @AfterAll
     public static void restoreSystemInput() {
-        System.setIn(DEFAULT_STDIN);
+        MatcherAssert.assertThat(
+            System.in,
+            Matchers.equalTo(DEFAULT_STDIN)
+        );
     }
 
+    @AfterEach
+    @ReadsStdIo
+    public void clearInput() {
+        final Input input = Input.getInstance();
+        try {
+            final Field prop = input.getClass().getDeclaredField("instance");
+            prop.setAccessible(true);
+            prop.set(input, null);
+        } catch (final NoSuchFieldException exception) {
+            exception.printStackTrace();
+            throw new RuntimeException(exception);
+        } catch (final IllegalAccessException exception) {
```

Comment:
@includealex I don't get this part at all. Why do you catch them here?

Reply:
@yegor256 if I remove catching of either IllegalAcessException or NoSuchFieldException I will get this error from compiler:
``` bash
Unhandled exception type IllegalAccessException
```
So this part of code can't be removed.

Reply:
@includealex then don't print stacktrace. Just catch and throw `IllegalStateException(ex)`